chrome/rhel-navigation.json

```diff
@@ -145,6 +145,24 @@
                             "product": "Red Hat Insights"
                         }
                     ]
+                },
+                {
+                    "title": "Malware",
+                    "expandable": true,
+                    "routes": [
+                        {
+                            "appId": "malware",
+                            "title": "Signatures",
+                            "href": "/insights/malware/systems",
```

Comment:
are both of these hrefs supposed to be the same?

Reply:
nope